atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/SweeperServiceImplTest.java

```diff
@@ -182,16 +183,12 @@ public void runsOneIterationIfRequested() {
         verifyNoMoreInteractions(sweepTaskRunner);
     }
 
-    private void verifyExpectedArgument(SweepResults observedResult) {
-        assertThat(observedResult.getTimeSweepStarted())
-                .isGreaterThanOrEqualTo(RESULTS_NO_MORE_TO_SWEEP.getTimeSweepStarted());
-        assertThat(observedResult.getTimeSweepStarted())
-                .isLessThanOrEqualTo(RESULTS_NO_MORE_TO_SWEEP.getTimeSweepStarted() + 5000L);
-        assertThat(RESULTS_NO_MORE_TO_SWEEP)
-                .isEqualTo(SweepResults.builder()
-                        .from(observedResult)
-                        .timeSweepStarted(RESULTS_NO_MORE_TO_SWEEP.getTimeSweepStarted())
-                        .build());
+    private void verifyExpectedArgument(long sweepTime, long totalTimeElapsed) {
+        assertThat(RESULTS_NO_MORE_TO_SWEEP.getTimeInMillis()).isEqualTo(sweepTime);
+        assertThat(totalTimeElapsed)
+                .isLessThanOrEqualTo(RESULTS_NO_MORE_TO_SWEEP.getTimeElapsedSinceStartedSweeping());
+        assertThat(totalTimeElapsed)
+                .isGreaterThanOrEqualTo(RESULTS_NO_MORE_TO_SWEEP.getTimeElapsedSinceStartedSweeping() - 1000L);
```

Comment:
nit: equivalent to `assertThat(totalTimeElapsed).isBetween(RNMTS.getTimeElapsed() - 1000, RNMTS.getTimeElapsed());`